DigitalLearningSolutions.Web/Views/VerifyEmail/Index.cshtml

```diff
@@ -0,0 +1,130 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.ViewModels.VerifyEmail
+@model VerifyEmailViewModel
+
+@{
+  ViewData["Title"] = "Verify your email address";
+}
+
+@section NavMenuItems {
+  <partial name="_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Verify your email address</h1>
+
+    @if (Model.UnverifiedEmailsCount == 0) {
+      <p class="nhsuk-body-m"> All your email addresses are verified.</p>
+    } else {
+      var centreEmailsExcludingFirstParagraph = (Model.PrimaryEmail == null
+        ? Model.CentreSpecificEmails.Skip(1)
+        : Model.CentreSpecificEmails).ToList();
+      const string centreEmailExplanation = "You will not be able to access that account until you verify the address.";
+
+      @if (Model.EmailVerificationReason.Equals(EmailVerificationReason.EmailNotVerified)) {
+        string primaryEmail;
+        string emailAddresses;
+        string link;
+        string address;
+        string email;
+        string folder;
+        string pronoun;
+
+        @if (Model.UnverifiedEmailsCount == 1) {
+          primaryEmail = null;
+          emailAddresses = Model.PrimaryEmail ?? "this address";
+          link = "link";
+          address = "address";
+          email = "this email";
+          folder = "folder";
+          pronoun = "it";
+        } else {
+          primaryEmail = $", {Model.PrimaryEmail},";
+          emailAddresses = "each of the addresses above";
+          link = "links";
+          address = "addresses";
+          email = "the emails";
+          folder = "folders";
+          pronoun = "them";
+        }
```

Comment:
I think I'd probably inline each of these to a ternary operator (?:) statement. I know it means that we're checking the condition multiple times, but it reduces the places you need to check what value is what. 
It might be worth defining a model property for `Model.UnverifiedEmailsCount == 1` to make it more clear?
(convert code style to var if you do this)

Reply:
We can also then move these variables closer to where they are actually used. Maybe even put all the ones that are only used in the `verifyEmailExplanation` directly into the string interpolation? Not really sure that is better though.

Reply:
I find having ternary operators inside string interpolations harder to follow, so I'll leave them as variables